src/modules/moteur.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] `required`
- [ ] Please enable disabled eslint because it defeats the purpose of installing linters 🥂